RealmWeaver/ModuleWeaver.cs

```diff
@@ -258,7 +260,12 @@ private void WeaveType(TypeDefinition type, Dictionary<string, Tuple<MethodRefer
         if (mapToAttribute != null)
             columnName = ((string) mapToAttribute.ConstructorArguments[0].Value);
 
+     //   Debugger.Launch();
+
         var backingField = GetBackingField(prop);
+        if (backingField == null) {
+            LogErrorPoint($"Property {prop.Name} has no backing field", sequencePoint);
```

Comment:
Ahhh, good catch, that was a debugging addition when I was trying to decipher the weird error messages from my IL being wrong.